SomerenUI/EditDrinkForm.cs

```diff
@@ -75,17 +60,21 @@ private void btnUpdateDrink_Click(object sender, EventArgs e)
             }
         }
 
-        private void ValidateInputs(out double tryGetPrice, out int tryGetStock)
+        private void LoadForm(string formName)
         {
-            if (txtDrinkName.Text.IsNullOrEmpty())
-                throw new Exception(Properties.Resources.ErrorMessageNoName);
-
-            tryGetPrice = double.Parse(txtDrinkPrice.Text.Replace(',', '.'), CultureInfo.InvariantCulture);
-
-            if (!int.TryParse(txtDrinkStock.Text, out int tryGetInt))
-                throw new Exception(Properties.Resources.ErrorMessageWrongStock);
+            if (formName.Equals(Properties.Resources.CreateDrink))
+            {
+                Text = formName;
```

Comment:
Text = formName; can be done before the if statement, so the code duplication could be avoided. Also, rdoTrue.Checked = true; is redundant because it's set in the designer.

private void LoadForm(string formName)
{
    Text = formName;

    if (formName.Equals(Properties.Resources.CreateDrink))
    {
        btnUpdateDrink.Hide();
    }
    else
    {
        btnCreateDrink.Hide();
        LoadText();
    }
}